documentation/assemblies/assembly-kafka-connect-authorization.adoc

```diff
@@ -0,0 +1,34 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-connect.adoc
+// assembly-deployment-configuration-kafka-connect-s2i.adoc
+
+[id='assembly-kafka-connect-authorization-{context}']
+
+= Kafka Connect user authorization
+
+If authorization is enabled for Kafka Connect, the `KafkaUser` resource for the Kafka Connect user must be configured to provide read/write access rights to the consumer group and
+internal topics, which are automatically configured or specified as configuration parameters in the `spec` for the `KafkaConnect` or `KafkaConnectS2I` configuration.
+
```

Comment:
_Suggested change:_

"...and internal topics. These access rights are automatically configured or specified in the `spec` for the `KafkaConnect` or `KafkaConnectS2I` configuration."

^ Does that change the meaning? I didn't quite understand what gets automatically configured: the access rights, or the consumer group and internal topics.

Reply:
I've tried to make it more clear:

> The properties for the consumer groups and internal topics are automatically configured, or specified in the `spec` for the `KafkaConnect` or `KafkaConnectS2I` configuration.

Reply:
That's much more understandable now. Thanks!